newm-server/src/main/kotlin/io/newm/server/features/distribution/eveara/EvearaDistributionRepositoryImpl.kt

```diff
@@ -1436,6 +1439,15 @@ class EvearaDistributionRepositoryImpl(
         log.info { "Distributed release ${songToRelease.title} with id ${songToRelease.distributionReleaseId} to future outlets: ${distributeFutureReleaseResponse.message}" }
     }
 
+    override suspend fun getEarliestReleaseDate(userId: UUID): LocalDate {
+        val maxDays = getOutlets(userRepository.get(userId)).outlets.maxOfOrNull { it.processDurationDates }
+            ?: throw HttpServiceUnavailableException("No Distribution Outlets available - retry later")
+        log.info { "Release process maximum days: $maxDays" }
```

Comment:
Remove this log or make it debug/trace. It's low-value to see in the logs.

Reply:
Removed.

Reply:
Removed.